### PR TITLE
refs: Remove mysql related code

### PR DESCRIPTION
**Pull request by wedamija**
Removes all code related to running mysql. Also removes uses of `.replace(microsecond=0)` where
appropriate, since most of the time that was used it was for mysql compatibility.

finn burn

### CODE REVIEW

File: src/sentry/db/models/fields/bounded.py [hunk 1]
```diff
@@ -71,9 +71,7 @@ class BoundedBigAutoField(models.AutoField):
 
         def db_type(self, connection):
             engine = connection.settings_dict['ENGINE']
-            if 'mysql' in engine:
-                return "bigint AUTO_INCREMENT"
-            elif 'oracle' in engine:
+            if 'oracle' in engine:
```
Comment (mattrobenolt) [inline src/sentry/db/models/fields/bounded.py]: Can we uhh, remove the oracle references too? I assume those came in from copy/paste.

Comment (wedamija) [reply src/sentry/db/models/fields/bounded.py]: Lol, sure

Comment (wedamija) [reply src/sentry/db/models/fields/bounded.py]: Removed a couple references to mssql as well, amazing

**Review by markstory — APPROVED** [review]
Looks good to me.

**Pull request merged**